pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "0.1.56"
+version = "0.1.58"
```

Comment:
I have taken a punt that my pull request with `0.1.58` will go in before this one.